src/identities/CommandClaim.ts

```diff
@@ -13,17 +13,17 @@ class CommandClaim extends CommandPolykey {
     this.name('claim');
     this.description('Claim a Digital Identity for this Keynode');
     this.argument(
-      '<gestaltId>',
-      '`Provider ID:Identity ID`',
-      binParsers.parseGestaltId,
+      '<providerIdentityId>',
+      'Name of the digital identity provider',
+      binParsers.parseGestaltIdentityId,
```

Comment:
You still want to be clear how it's formatted here.

Reply:
Line 17 use the string that trust/discover uses.